src/NextjsLambda.ts

```diff
@@ -22,6 +22,12 @@ function getEnvironment(props: NextjsLambdaProps): { [name: string]: string } {
     ...props.environment,
     ...props.lambda?.environment,
     ...(props.nodeEnv ? { NODE_ENV: props.nodeEnv } : {}),
+    ...{
+      CACHE_BUCKET_NAME: props.cacheBucket?.bucketName || '',
+      CACHE_BUCKET_REGION: props.cacheBucket ? Stack.of(props.cacheBucket).region : '',
+      // Note we don't need a CACHE_BUCKET_KEY_PREFIX because we're using a separate bucket for cache
```

Comment:
I'm not against creating another bucket for the cache, but why not reuse static asset bucket with _cache prefix like SST? The smaller footprint for this construct, the better, right?

Reply:
Is there any possibility you could have cached data that you would not want to be publicly accessible? Can people have internal pages cached for example? Would the filenames be guessable?

Reply:
I originally was actually using `_cache`, [but the `open-next` recommended architecture](https://github.com/serverless-stack/open-next/blob/main/README.md) has them as separate buckets. I sort of like the idea of being able to define separate behavior for these buckets. That said, I did realize SST seems to use the same bucket. I don't have enough experience with the trade-offs here to have a strong opinion, but those were my reasons.

Reply:
@revmischa, I don't think so, because the point of the cache is to make pages ready that are actually being accessed by users. It wouldn't make sense to cache pages that aren't being served by users.

Reply:
@kevin-mitchell, I think the open-next diagram's duplicate bucket is only for simplicity of displaying. I'd prefer to same bucket because I think it semantically makes sense for the cached assets to be in the assets bucket but I'm not hard pressed on this opinion.

Reply:
Their language [here](https://d328emyhtjj3hd.cloudfront.net) (emphasis mine) to me at least makes it sound like re-using the same bucket is the norm:

> Typically, asset files are uploaded to the root of the bucket. However, you **might** want to store them in a subfolder of the bucket, for instance, when:
> using a pre-existing bucket; or
> storing both assets and cache files in the same bucket.

To me, "cache" === "can be destroyed without breaking things", vs the "assets" directory is genuinely static as far as I can tell - it's more of a "static deployment artifact" bucket. 

(Hopefully this doesn't come off as argumentative. I'm 100% just a guest here and will happily modify this, and really don't have any strong feelings on this. It's possibly I'm partially motivated by the fact that I'll be away from a computer for the next few days and might not get a chance to update and test changing things - the main impact I can see is if we use the sub-directory in the single bucket then the "rewriter" comes into play as it acts on those static files / paths in the assets bucket I believe. Also, I'm not 100% sure how multiple deployments to the same bucket impact each other. None of this is "hard" exactly, just not sure I'll get to it until next week).

Reply:
@kevin-mitchell, no worries at all! I'm working on using the same bucket. I don't think it will impact the rewriter. I'll make sure it works with the deployment construct.